config.go

```diff
@@ -732,6 +732,8 @@ func NewInclude(directives []string, hasEquals bool, pos Position, comment strin
 			path = directives[i]
 		} else if system {
 			path = filepath.Join("/etc/ssh", directives[i])
+		} else if strings.HasPrefix(directives[i], "~/") || strings.HasPrefix(directives[i], "~\\") {
+			path = filepath.Join(homedir(), directives[i][2:])
```

Comment:
Is `2:` right here? Wouldn't it be different in the first condition (only 2 chars) vs. the second condition (3 chars) ?

Reply:
The second one is 2 chars. The first `\` of `\\` is the escape char for `\`. `"~\\"` in source code means `~\`.